fabric-convention-tags-v1/src/main/java/net/fabricmc/fabric/api/tag/convention/v1/ConventionalEntityTypeTags.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.convention.v1;
+
+import net.minecraft.entity.EntityType;
+import net.minecraft.tag.TagKey;
+
+import net.fabricmc.fabric.impl.tag.convention.TagRegistration;
+
+/**
+ * See {@link net.minecraft.tag.EntityTypeTags} for vanilla tags.
+ * Note that addition to some vanilla tags implies having certain functionality.
+ */
+public class ConventionalEntityTypeTags {
```

Comment:
Can you add a tag for custom item entity subclasses that should not be picked up by magnets? (e.g. AE2 seeds / botania mana tablets) Or is this too narrow?

Reply:
Actually, it would make more sense for it to be an item tag, not an entity type tag!

Reply:
IMO it'd be better to not add any more applications to this PR and do them separately

Reply:
Sure. These don't have any value by default, so mods can support them separately. We only need them in the API if mods can't agree on a standard id.

Reply:
If needed, this can be added later.